packages/forms/resources/views/components/rich-editor.blade.php

```diff
@@ -458,7 +458,8 @@ class="trix-dialog trix-dialog--link"
                                     name="href"
                                     placeholder="{{ __('filament-forms::components.rich_editor.dialogs.link.placeholder') }}"
                                     required
-                                    type="url"
+                                    type="text"
+                                    pattern = "(https?://|/).+"
```

Comment:
```suggestion
                                    inputmode="url"
```

Doing this because relative links don't even need to start with a /, and this attribute puts the specialised keyboard back for mobile devices